docs/table/access_table.rst

```diff
@@ -86,6 +86,7 @@ For all the following examples it is assumed that the table has been created as
 
   >>> from astropy.table import Table, Column
   >>> import numpy as np
+  >>> import astropy.units as u
```

Comment:
Yikes, that mangled sentence has been there for 3.5 years.  I guess nobody is reading the docs...